src/js/index.js

```diff
@@ -0,0 +1,157 @@
+import { $ } from "./util/helper.js";
+
+const $menuCount = $(".menu-count");
+
+const $menuForm = $("#menu-form");
+const $menuNameInput = $("#menu-name");
+
+const $menuList = $("#menu-list");
+
+const $nav = $("nav");
+const $categoryTitle = $("#category-title");
+
+const store = {
+  setLocalStorage(value) {
+    localStorage.setItem("menu", JSON.stringify(value));
+  },
+
+  getLocalStorage() {
+    return JSON.parse(localStorage.getItem("menu"));
+  },
+};
```

Comment:
```suggestion
export const store = {
  setLocalStorage(value) {
    localStorage.setItem("menu", JSON.stringify(value));
  },

  getLocalStorage() {
    return JSON.parse(localStorage.getItem("menu"));
  },
};
```
한 파일에 하나의 컴포넌트만 있게 구성하는게 좋다!